lib/vaahextendflutter/base/base_controller.dart

```diff
@@ -54,8 +62,10 @@ class BaseController extends GetxController {
             ..dsn = config.sentryConfig!.dsn
             ..autoAppStart = config.sentryConfig!.autoAppStart
             ..tracesSampleRate = config.sentryConfig!.tracesSampleRate
-            ..enableAutoPerformanceTracing = config.sentryConfig!.enableAutoPerformanceTracing
-            ..enableUserInteractionTracing = config.sentryConfig!.enableUserInteractionTracing
+            ..enableAutoPerformanceTracing =
+                config.sentryConfig!.enableAutoPerformanceTracing
+            ..enableUserInteractionTracing =
+                config.sentryConfig!.enableUserInteractionTracing
```

Comment:
make sure your code editor has 100 character line break not 80.

Reply:
Done with this can you give some extension for imports for Android Studio, am not using VS code.

Reply:
hmm, install vs code and that extension and format all the imports when needed, use android studio otherwise.